draft-ietf-scitt-architecture.md

```diff
@@ -673,11 +673,10 @@ This is a decoded inclusion proof for RFC9162_SHA256, other verifiable data stru
 
 ### Validation {#validation}
 
-Verifiers MUST apply the verification process as described in Section 4.4 of RFC9052.
+Verifiers MUST apply the verification process as described in Section 4.4 of {{RFC9052}}.
 
-In order to verify the inclusion proof that is included in the Receipt, the verification process for the inclusion proof MUST be performed as described in the document that registers corresponding Verifiable Data Structure Parameters (see {{-COMETRE}}).
-
-APIs exposing verification logic for Transparent Statements may wish to provide more details that a single boolean result, for example, indicating if the signature on the Receipt or Signed Statement is valid, if claims related to the validity period are valid, or if the inclusion proof in the Receipt is valid.
+APIs exposing verification logic for Transparent Statements may wish to provide more details than a single boolean result.
+For example, indicating if the signature on the Receipt or Signed Statement is valid, if claims related to the validity period are valid, or if the inclusion proof in the Receipt is valid.
```

Comment:
The above two lines are simply separated sentences for clarity, no meaningful change.
Line [678](https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/pull/208/files#diff-585fb424519b79cf000445f3425fa56d328cbaca5d2a622740ddc4e5ca91dbe1L678) was removed as it's been edited to a lack of clarity, and redundant with line [225](https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/pull/208/files#diff-585fb424519b79cf000445f3425fa56d328cbaca5d2a622740ddc4e5ca91dbe1L225)

Reply:
I agree, but would prefer "For example, an API may indicate if ..." to avoid a nominal sentence.